roles/engineblock/tasks/main.yml

```diff
@@ -64,6 +64,30 @@
   when:
     customlogo.stat.exists
 
+- name: Check if we have a custom background back image for the feedback page
+  local_action: stat path="{{ inventory_dir }}/files/eb/background-back.svg"
+  register: eb_customfeedbackbackground
+  become: false
+
+- name: Install environment specific background back image
```

Comment:
Proposal to improve: 

> Install environment specific background back image

> Set the environment specific error feedback page background image. (background)

The same suggestion goes for the next taskname, but instead of `(background)` set `(foreground)`

Reply:
It was shamefully pasted from the tasks above and I think it will be sufficient as-is.